src/Model/Release/LibraryRelease.php

```diff
@@ -230,4 +238,27 @@ public function setChangelog($changelog)
         $this->changelog = $changelog;
         return $this;
     }
+
+    /**
+     * Get branching strategy
+     *
+     * @param string $default Default value to get if no value found
+     * @return string
+     */
+    public function getBranching($default = Branch::DEFAULT_OPTION)
```

Comment:
Do you need to have this as an argument, since it's a default and a constant already? E.g. could you just return the constant if the class property isn't set?

Reply:
The reason we need the $default is so that we can re-set it to `null` when serialising.

See https://github.com/silverstripe/cow/pull/59/files#diff-71b76cc058aab67051fd16cc5a6dc6d9R994

Reply:
Cool

Reply:
Do we need the `DEFAULT_OPTION` constant? Seems a bit redundant to define it at this stage